fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/EraseResourceDAO.java

```diff
@@ -121,7 +122,7 @@ private void runCallableStatement(String callStr, long erasedResourceGroupId) th
      * @param erasedResourceGroupId
      * @throws SQLException
      */
-    public void runInDao(long erasedResourceGroupId) throws SQLException {
+    public void runInDao(long erasedResourceGroupId) throws SQLException, FHIRPersistenceException {
```

Comment:
Maybe this should now throw only FHIRPersistenceException or DataAccessException?

Reply:
done